docs/installation/mfa_integration.asciidoc

```diff
@@ -0,0 +1,127 @@
+// to display images directly on GitHub
+ifdef::env-github[]
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+:imagesdir: ../images
+endif::[]
+
+////
+
+    This file is part of the PacketFence project.
+
+    See PacketFence_Installation_Guide.asciidoc
+    for authors, copyright and license information.
+
+////
+
+//== MFA Integration
+
+This section has been created to give a quick overview to configure MFA integration with PacketFence.
+
+=== Assumptions
+
+You have a fonctional PacketFence server and you configured an Internal Source (like Active Directory Source) associated to a "Connection Profile"
+You also have a radius client that is doing PAP (like a VPN server or a switch with CLI access enabled to use RADIUS). 
+
+=== Create the MFA Configuration
+
+[[akamai-mfa]]
+==== Akamai MFA
+
+include::mfa/akamai.asciidoc[]
+
+[[totp-mfa]]
+==== TOTP MFA
+
+include::mfa/totp.asciidoc[]
+
+
+==== Associate the source 
+
+The MFA is triggered by a authentication rule in the Internal Source.
+You have to create a rule with a condition like "memberof equals cn=otp_user,dc=acme,dc=com" and assign a Action:
+
+"Trigger RADIUS MFA" if you want to be triggered in RADIUS
+"Trigger Portal MFA" if you want to be triggered in the Portal.
+
+image::mfa_rule.png[scaledwidth="100%",alt="MFA Authentication rule"]
+
+==== Portal Flow
+
+Depending of the MFA provider you use, the portal flow will be different.
+
+===== Akamai Bind v2
+
+include::mfa/akamai_bind_v2.asciidoc[]
+
+===== TOTP
+
+include::mfa/totp_portal.asciidoc[]
+
+==== RADIUS Flow
+
+The RADIUS flow depends of the feature of the MFA provider and also depends of the radius client.
+
+===== Simple RADIUS client
+
+In this use case only the username and password is sent in the RADIUS request, the only method available is the "push" notification.
+Once the user authenticate, he receive a push notification on his phone and have to validate it in order to grant the access.
```

Comment:
Once the user authenticate, he receive a push notification -> Once the user authenticated, a push notification will be sent

and have -> and the user will have

to grant the -> to be granted